cabal.project

```diff
@@ -5,3 +5,15 @@ packages: microlens/
           microlens-platform/
           microlens-th/
           microlens-pro/
+
+tests: True
+test-show-details: direct
```

Comment:
The `tests: True` makes the tests compile by default instead of having to add `--enable-tests`.

The `test-show-details: direct` make the test output shown on `stdout` instead of just being output to a file.

Reply:
Feel free to delete them if you want.